modules/shared/programs/claude/files/skills/plan-with-questions/references/consulting-step.md

```
@@ -203,13 +325,79 @@ rm -rf -- "$CONSULT_DIR"
 
 자문 결과를 `AskUserQuestion`(또는 등가 도구)으로 제시할 때 다음을 모두 적용:
 
-1. **"(Recommended)" 라벨 금지** — `option.label`에 추천 표시 안 함. 자문 출력에 score/rank가 있어도 무시.
+1. **추천 라벨 — "허용 + 합의 조건" (D4 합의 알고리즘)** — `(Recommended)` 라벨은 **합의 PASS 단일 옵션에만 부착**한다. 자문 결과 score/rank 같은 implicit choice 필드는 무시 — 라벨 부착 결정은 메인 LLM의 합의 알고리즘만 수행한다. AskUserQuestion 도구 description이 라벨을 권장해도 본 hard rule이 우선한다.
+
+   **D4 합의 알고리즘 4단계** (사용자 노출 직전 메인 LLM이 결정마다 실행 — schema 한계 내 보수적 합의 정의):
+
+   ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)